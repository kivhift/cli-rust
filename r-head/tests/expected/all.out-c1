==> tests/inputs/UTF-8-demo.txt <==


==> tests/inputs/empty.txt <==

==> tests/inputs/one.txt <==
Ã
==> tests/inputs/two.txt <==
T
==> tests/inputs/three.txt <==
T
==> tests/inputs/ten.txt <==
o