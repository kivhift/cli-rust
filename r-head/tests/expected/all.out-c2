==> tests/inputs/UTF-8-demo.txt <==

U
==> tests/inputs/empty.txt <==

==> tests/inputs/one.txt <==
Ã–
==> tests/inputs/two.txt <==
Tw
==> tests/inputs/three.txt <==
Th
==> tests/inputs/ten.txt <==
on