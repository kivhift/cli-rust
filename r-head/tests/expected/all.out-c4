==> tests/inputs/UTF-8-demo.txt <==

UTF
==> tests/inputs/empty.txt <==

==> tests/inputs/one.txt <==
Ã–ne
==> tests/inputs/two.txt <==
Two 
==> tests/inputs/three.txt <==
Thre
==> tests/inputs/ten.txt <==
one
