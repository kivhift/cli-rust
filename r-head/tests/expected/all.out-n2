==> tests/inputs/UTF-8-demo.txt <==

UTF-8 encoded sample plain-text file

==> tests/inputs/empty.txt <==

==> tests/inputs/one.txt <==
Ã–ne line, four words.

==> tests/inputs/two.txt <==
Two lines.
Four words.

==> tests/inputs/three.txt <==
Three
lines,

==> tests/inputs/ten.txt <==
one
two
