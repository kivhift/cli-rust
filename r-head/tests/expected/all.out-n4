==> tests/inputs/UTF-8-demo.txt <==

UTF-8 encoded sample plain-text file
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾


==> tests/inputs/empty.txt <==

==> tests/inputs/one.txt <==
Öne line, four words.

==> tests/inputs/two.txt <==
Two lines.
Four words.

==> tests/inputs/three.txt <==
Three
lines,
four words.

==> tests/inputs/ten.txt <==
one
two
three
four
