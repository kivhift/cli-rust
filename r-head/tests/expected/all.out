==> tests/inputs/UTF-8-demo.txt <==

UTF-8 encoded sample plain-text file
‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾

Markus Kuhn [ˈmaʳkʊs kuːn] <http://www.cl.cam.ac.uk/~mgk25/> — 2002-07-25


The ASCII compatible UTF-8 encoding used in this plain-text file
is defined in Unicode, ISO 10646-1, and RFC 2279.


==> tests/inputs/empty.txt <==

==> tests/inputs/one.txt <==
Öne line, four words.

==> tests/inputs/two.txt <==
Two lines.
Four words.

==> tests/inputs/three.txt <==
Three
lines,
four words.

==> tests/inputs/ten.txt <==
one
two
three
four
five
six
seven
eight
nine
ten
